packages/drivers/odsp-driver-definitions/src/factory.ts

```diff
@@ -89,4 +89,9 @@ export interface HostStoragePolicy {
     // True to have the sharing link redeem fallback in case the Trees Latest/Redeem 1RT call fails with redeem error.
     // During fallback it will first redeem the sharing link and then make the Trees latest call.
     enableRedeemFallback?: boolean;
+
+    /**
+     * Policy controlling if createNewFluidFile will cache initial summary
```

Comment:
Remove the "createNewFluidFile" function name from comment here as it is not useful for host what function name we are using to cache that summary. Just describe what is it for.